### PR TITLE
Split Provinzial into two companies

### PR DESCRIPTION
**Pull request by tordans**
Fixes https://github.com/osmlab/name-suggestion-index/issues/4544

Update: ID is auto-generated now.

### CODE REVIEW

**Comment by tordans** [comment]
Question: When I ran the build-script, the script re-added the previous data again. I dont know where that came from, since I removed it (twice). I did not commit the previous data, just the new one. Should I?

File: data/brands/office/insurance.json [hunk 1]
```diff
@@ -746,12 +746,26 @@
...
-      "id": "provinzial-422e71",
+      "displayName": "Provinzial NordWest",
+      "id": "provinzialnordwest-422e71",
       "locationSet": {"include": ["de"]},
       "tags": {
-        "brand": "Provinzial",
-        "name": "Provinzial",
+        "brand": "Provinzial NordWest",
+        "brand:wikidata": "Q1778922",
+        "brand:wikipedia": "de:Provinzial_NordWest",
```
Comment (bhousel) [inline data/brands/office/insurance.json:755]:
```suggestion
        "brand:wikipedia": "de:Provinzial NordWest",
```

File: data/brands/office/insurance.json [hunk 2]
```diff
@@ -746,12 +746,26 @@
...
+      }
+    },
+    {
+      "displayName": "Provinzial Rheinland",
+      "id": "provinzialrheinland-422e71",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "brand": "Provinzial Rheinland",
+        "brand:wikidata": "Q2114066",
+        "brand:wikipedia": "de:Provinzial_Rheinland",
```
Comment (bhousel) [inline data/brands/office/insurance.json:767]:
```suggestion
        "brand:wikipedia": "de:Provinzial Rheinland",
```

**Review by bhousel — CHANGES_REQUESTED** [review]
Thanks @tordans , looks mostly good!
The wikidata tags need underscores replaced with spaces, then good to merge..

**Comment by tordans** [comment]
> The wikidata tags need underscores replaced with spaces, then good to merge..

@bhousel thank for the change-suggestions, I commited them.
Where would I normally copy this string from? Is it the title of the wikipedia-doc prefixed with the lang?

**Comment by Identitaet** [comment]
You'll find the language code at the begging of the Wikipedia link something like 
`www.en.wikpedia and so on`
And the non encoded version of the name will be in the top left as the title

**Review by bhousel — APPROVED** [review]

**Pull request merged**